src/style.css

```diff
@@ -1,3 +1,67 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  margin: 0;
+  height: 60px;
+  border: 1px solid rgb(175, 174, 174);
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav--link.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.logo {
+  padding: auto;
+  height: 40px;
+  width: 419px;
+  margin-top: auto;
+  margin-block: auto;
+}
+
+.link {
+  text-align: center;
+  max-height: 18px;
+  color: #00acdc;
+  min-height: 18px;
```

Comment:
The `.link` class has both `max-height` and `min-height` set to 18px. This is redundant and can be simplified to just `height: 18px;`.